runtask.yml

```diff
@@ -12,13 +12,19 @@ environment:
         - pdc-client
         - python2-aexpect
         - python2-avocado
+        - python2-avocado-plugins-varianter-yaml-to-mux
```

Comment:
This plugin is currently available only in the Fedora update-testing repos, so the package needs to be installed via a separate shell action by running  `dnf -y --enablerepo=updates-testing install python2-avocado-plugins-varianter-yaml-to-mux`.